src/main/io/serial_msp.c

```diff
@@ -750,6 +750,13 @@ static bool processOutCommand(uint8_t cmdMSP)
         serialize16(constrain(averageSystemLoadPercent, 0, 100));
         break;
 
+    case MSP_NAME:
+        headSerialReply(MAX_NAME_LENGTH);
```

Comment:
You could actually send the length of the null terminated string here or max_name_length which ever is the shorter, and then only send the bytes needed - that way if we ever extend the length the configurator and firmware are not tied together.